gcsio/src/test/java/com/google/cloud/hadoop/gcsio/integration/GoogleCloudStorageImplTest.java

```diff
@@ -248,28 +253,31 @@ public void conflictingWrites_noOverwrite_lastFails() throws IOException {
 
     assertThat(trackingGcs.requestsTracker.getAllRequestStrings())
         .containsExactly(
-            getRequestString(resourceId.getBucketName(), resourceId.getObjectName()),
             getRequestString(resourceId.getBucketName(), resourceId.getObjectName()),
             resumableUploadRequestString(
                 resourceId.getBucketName(),
                 resourceId.getObjectName(),
                 /* generationId= */ 1,
                 /* replaceGenerationId= */ true),
-            resumableUploadRequestString(
+            resumableUploadChunkRequestString(
                 resourceId.getBucketName(),
                 resourceId.getObjectName(),
                 /* generationId= */ 2,
-                /* replaceGenerationId= */ true),
-            resumableUploadChunkRequestString(
+                /* replaceGenerationId= */ 1));
+
+    assertThat(trackingGcs2.requestsTracker.getAllRequestStrings())
+        .containsExactly(
+            getRequestString(resourceId.getBucketName(), resourceId.getObjectName()),
+            resumableUploadRequestString(
                 resourceId.getBucketName(),
                 resourceId.getObjectName(),
-                /* generationId= */ 3,
-                /* uploadId= */ 1),
+                /* generationId= */ 1,
+                /* replaceGenerationId= */ true),
             resumableUploadChunkRequestString(
                 resourceId.getBucketName(),
                 resourceId.getObjectName(),
-                /* generationId= */ 4,
-                /* uploadId= */ 2));
+                /* generationId= */ 2,
+                /* replaceGenerationId= */ 1));
```

Comment:
Here should be `uploadId` in the comment

Reply:
Updated